client/my-sites/github-deployments/components/repositories/deployment-style/deployment-style.tsx

```diff
@@ -432,11 +447,32 @@ export const DeploymentStyle = ( {
 
 			{ advancedOptionsDisabled && (
 				<FormInputValidation
+					className="advanced-options-error"
 					isError
-					text={ translate( "This repository template doesn't have workflows." ) }
+					text={ translate( 'The selected template does not support advance deployment.' ) }
 				/>
 			) }
 
+			{ isTemplateRepository && selectedWorkflow.workflow_path !== 'none' && (
+				<FormFieldset className="github-deployments-deployment-style__workflow-checks workflow-content">
+					<FoldableCard
+						expanded={ true }
+						header={ selectedWorkflow.workflow_path }
+						screenReaderText="More"
```

Comment:
This is already in the component so doesn't need to be added, but if it is included, might need to be translatable?

```suggestion
```